app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -163,4 +174,18 @@ def textual_network_devices
   def firmware_details
     @record.hardware.firmwares.collect { |fw| [fw.name, fw.version] }
   end
+
+  private
+
+  def create_https_url(ip)
+    url = ""
+
+    unless ip.nil?
+      # A target argument with a value of "_blank" is passed so that the
+      # page loads in a new tab when the link is clicked.
+      url = link_to(ip, "https://#{ip}", :target => "_blank")
+    end
+
+    url
+  end
```

Comment:
```
def create_https_url(id)
  ip.present? ? link_to(ip, "https://#{ip}", :target => "_blank") : ''
end
```
or
```
def create_https_url(id)
  ip.present? ? link_to(ip, URI::HTTP.build(:host =>ip), :target => "_blank") : ''
end
```

Reply:
@martinpovolny I tried using the URI::HTTPS.build method, but it's causing the following error:

`undefined method 'to_model' for #<URI::HTTPS https://10.243.6.68> [physical_server/show] `

Reply:
@martinpovolny Never mind, I got past the error by calling the to_s method on the object that's returned by the build method.